AP Table Update

woodbe opened the pull request:
This updates the AP table and the text to the newer table. The one problem I found is that the table does not include the Window of Opportunity (Access to the TOE) which we need. The Degree of Scrutiny isn't sufficient for this use case.

I am proposing that as an addition to the table. It is possible they could be merged, but I don't think so off hand. They are sort of exclusive to each other.

woodbe committed:
APTable

woodbe committed:
degree of scrutiny

woodbe committed:
more udpates

gregott commented:
I am not sure what you mean by "flexible support adaptation" in Knowledge of TOE > Equipment > Bespoke equipment (line 1571)

In the same paragraph, what is meant by "limited time access"?

gregott commented:
In Window of opportunity (Access to the TOE) > Non-cooperative, Do we want to add a sentence that the subject is not aware his biometric sample is being collected?   Also, what is the difference between "without full cooperation" and no cooperation?

woodbe commented:
@woodbe will rewrite limited time access in like 1571

woodbe commented:
@woodbe line 1587 replace "full" with "conscious" for cooperation.

woodbe commented:
@woodbe edit line 1601 - this should be no need to access the TOE, not the biometric characteristic

woodbe commented:
@woodbe  line 1608 fix authentication

woodbe commented:
@woodbe keep both Access to TOE and Degree of scrutiny. Make sure the explanatory text does not overlap given they are separate. Access to TOE descriptions will need to be adjusted.

woodbe commented:
@woodbe add proposed rows from ISO comments

woodbe commented:
@woodbe & @n-kai will look at "flexible support adaptation" text to see if we can come up with something better and clear.

woodbe committed:
July8 call update

n-kai commented:
> @woodbe & @n-kai will look at "flexible support adaptation" text to see if we can come up with something better and clear.

Attak potential chapter was developped based on the CC smart card supporting document (Application of Attack Potential to Smartcards and Simlar devices *1). I rephraised the para that includes "flexible support adaptation" and "limited time access" refering this document. 

_Bespoke equipment refers to very expensive equipment, the equipment is so specialized that its distribution is controlled, or the use of equipment requires either a very long experience or the human operator is hired together with the equipment; for example, the high-grade printing systems with the human operator hired together. In addition, if more than one specialized equipment is required to perform different parts of the attack, this value should be used. Before using this level, it shall be carefully checked the possibilities of renting equipment. If such service exists, the level shall be moved down to specialized level._

*1) https://www.commoncriteriaportal.org/files/ccfiles/JIL-Application-of-Attack-Potential-to-Smartcards-v3.2.1.pdf

gregott commented:
The rewording from @n-kai is great and makes the statements in question much more understandable.  Below is a version that is slightly "wordsmithed" for readability.

_Bespoke equipment refers to very expensive equipment. The equipment is so specialized that its distribution is controlled, or the use of equipment requires either significant experience to master its operation or a human operator is hired together with the equipment; for example, high-grade printing systems with a human operator hired together. In addition, if more than one specialized equipment is required to perform different parts of the attack, this value should be used. Before using this level, the possibility of renting equipment should be determined. If such a service exists, the level shall be moved down to the specialized level._

woodbe committed:
bespoke equipment

woodbe commented:
OK, I added the text from @gregott into the latest commit I just pushed.

woodbe committed:
Rating Table update

woodbe commented:
I just updated the rating table to match what is in the proposed ISO document. There are some differences as we initially had AVA_VAN.1/2 as meeting the same thing whereas the updated table has those split out but has the AVA_VAN.4/5 as both being met at the =>40 level. I updated the table to reflect this change.

n-kai committed:
Update BS_SD.adoc

n-kai committed:
Update BS_SD.adoc

n-kai reviewed (changes requested):
See my pull request #441

woodbe committed:
updated enhanced-easy

woodbe committed:
edits from call

woodbe committed:
Merge pull request #441 from biometricITC/n-kai-updates-to-ap-table

Update on Brian's proposal

n-kai reviewed (approved):
I will also update the table in ISO/IEC 19989-1 as described in this PR but the table in ISO may be modified/updated further based on comments from JWG7 experts. 
I will also add some examples of attack potential caluculation to ISO/IEC 19989-1 refering the attacks in the toolbox.
